src/providers/BuyDomainProvider.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from 'react';
+
+import { createContext, useState } from 'react';
+
+export const BuyDomainContext = createContext({
+	domainName: '',
+	setDomainName: (domainName: string) => {},
+});
```

Comment:
It could be nice to change the naming of these, as they currently aren't very descriptive.

- `domainName` - could be `selectedDomain` or similar.
- `setDomainName` - could be `selectDomain` or similar.

The Context and Provider names could be changed to `SelectDomain...` or similar, because the Context/Provider doesn't actually have any functionality to do with buying a domain - at the moment they just select a domain.

I'm not sure if `select` is the right word to use here, but I think we need something more descriptive than what's there currently.

Reply:
Updated.